consensus/service/src/mint_tx_manager/mod.rs

```diff
@@ -366,7 +366,7 @@ mod mint_config_tx_tests {
         let parent_block = ledger.get_block(n_blocks - 1).unwrap();
 
         let block_contents = BlockContents {
-            outputs: vec![create_test_tx_out(&mut rng)],
+            outputs: vec![create_test_tx_out(BlockVersion::MAX, &mut rng)],
```

Comment:
Consider adding a `const BLOCK_VERSION = BlockVersion::MAX` to make it obvious we're using the same block version for all these calls.